hydra.c

```diff
@@ -237,13 +237,7 @@ hydro_ngbiter(
     {
         double Pressure_j = PressurePred(other);
         double p_over_rho2_j = Pressure_j / (SPHP(other).EOMDensity * SPHP(other).EOMDensity);
-        double soundspeed_j;
-
-#ifdef DENSITY_INDEPENDENT_SPH
-        soundspeed_j = sqrt(GAMMA * Pressure_j / SPHP(other).EOMDensity);
-#else
-        soundspeed_j = sqrt(GAMMA * p_over_rho2_j * SPHP(other).Density);
-#endif
+        double soundspeed_j = sqrt(GAMMA * Pressure_j / SPHP(other).EOMDensity);
```

Comment:
This looks a bit suspicious -- doesn't the removal of ifdef break the density-energy sph force?

Reply:
Did I misread it? I think the two expressions are the same, because EOMDensity is EgyWtDensity for DENSITY_INDEPENDENT_SPH and Density otherwise.

Reply:
That's right -- I still remember there was some trickiness here. Perhaps it is integrated into pressurepred.